pkgs/native_assets_cli/lib/src/target.dart

```diff
@@ -5,8 +5,8 @@
 import 'dart:ffi' show Abi;
 import 'dart:io';
 
-import '../api/architecture.dart';
-import '../api/os.dart';
+import 'architecture.dart';
+import 'os.dart';
 
 final class Target implements Comparable<Target> {
```

Comment:
I believe this should stay in model/ it's never used in hooks and not part of the API. It's only used in `native_assets_builder`.

Reply:
This is actually used in flutter tools, see https://github.com/flutter/flutter/blob/f7b363929d897800db59137a5c5de443c1f28bed/packages/flutter_tools/lib/src/isolated/native_assets/android/native_assets.dart#L55

@dcharkes Do you prefer us getting rid of the usage in flutter tools?

Reply:
Flutter tools can import `internal.dart`, the build runner takes `Target` as an argument. We could opt to change that to `OS` and `Architecture`. Whatever you prefer.

Reply:
Moved it to `..._internal.dart`